SPECS/glibc/glibc.spec

```diff
@@ -174,24 +170,24 @@ rm -rf %{buildroot}%{_infodir}
 cat > %{buildroot}%{_sysconfdir}/nsswitch.conf <<- "EOF"
 #       Begin /etc/nsswitch.conf
 
-    passwd: files
-    group: files
-    shadow: files
+	passwd: files
```

Comment:
Why not just remove the whitespaces completely?

Reply:
I just restored it to how it was in the 1.0 branch.